tools/runners/run-unittests.sh

```diff
@@ -59,14 +59,13 @@ UNITTEST_TEMP=`mktemp unittest-out.XXXXXXXXXX`
 
 for unit_test in $UNITTESTS
 do
-    echo -n "[$tested/$total] ${unit_test#$ROOT_DIR}: "
-
+    cmd_line="${unit_test#$ROOT_DIR}"
     $unit_test &>$UNITTEST_TEMP
     status_code=$?
 
     if [ $status_code -ne 0 ]
     then
-        echo "FAIL ($status_code)"
+        echo "[$tested/$total] $cmd_line: FAIL ($status_code)"
```

Comment:
I wanted to keep the same pattern in both runner scripts, and also didn't want to duplicate the path strip logic (`#$ROOT_DIR`) to help maintainability. Both it's no big code so we can merge this if you think so.